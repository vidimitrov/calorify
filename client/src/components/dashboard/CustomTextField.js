import styled from 'styled-components';
import TextField from '@material-ui/core/TextField';

const CustomTextField = styled(TextField)`
  width: 250px;
  margin: 20px;
`;

export default CustomTextField;
