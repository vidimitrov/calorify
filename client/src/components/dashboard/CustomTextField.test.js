import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CustomTextField from './CustomTextField';

it('Dashboard/CustomTextField component renders correctly', () => {
  const tree = renderer.create(<CustomTextField value="some-value" />).toJSON();
  expect(tree).toMatchSnapshot();
});
