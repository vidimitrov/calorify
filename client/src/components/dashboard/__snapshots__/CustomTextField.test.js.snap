// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/CustomTextField component renders correctly 1`] = `
.c0 {
  width: 250px;
  margin: 20px;
}

<div
  className="MuiFormControl-root-1 c0"
>
  <div
    className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-28 MuiInput-input-13"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
      value="some-value"
    />
  </div>
</div>
`;
